json/json-quiz.md

```diff
@@ -580,3 +580,24 @@ printNullness(JSON.parse('{ "lemmings": "null" }'));
 - `->`
 - `::`
 - `.` <<<<---- Correct
+
+#### Q48. If a string contains line breaks, what shhould you do?
+
+- Wrap line breaks in single quotation marks
+- Make no changes, because whitespace is allowewd
+- Replace any line breaks with `\r`
+- Replace any line breaks with `\n` <<<<---- Correct
+
+#### Q49. Which number type are available in javascript but not supported in json?
```

Comment:
Typo? It's either 
`type` -> `types` because 'Which number types are available..' OR
`are` -> `is` because 'Which number type is available..'

Reply:
Updated it.